Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,37 @@
 
 ## Exercise I
 
-a)
+a) `O(n^3)`
+    Because while the while loop is (`a` is less than `n^3`), the runtime scales by `n` cubed
 
+b) **?** `O(2^n)`
+    What is happening here is `j` is being multiplied by 2 `n` number of times so the runtime is `O(2^n)`
 
-b)
+c) `O(n + 1)`
+    The function will run as many times as there are `bunnies` passed into it (plus one for the 0 `bunnies` base case)
 
+## Exercise II
 
-c)
+Divide the number of floors `n` by half (minus 1 if odd) and drop 1 egg. If it breaks, the floor is either too high or floor `f`, divide the bottom half of floors by 2 and drop again. If it doesn't break, it's too low, divide the top half by 2 and try again off the center floor. This method is a binary search algorithm and I believe should yield the most efficient results.
```

Comment:
Great explanation here. In problems like this it's always good to lean towards being as explicit as possible.